styles.css

```diff
@@ -0,0 +1,125 @@
+
+.mobile-header {
+  position: absolute;
+  width: 375px;
+  height: 92px;
```

Comment:
- [ ] Please get rid of all the width and height fixed values, the essence of this exercise is to make a responsive web design, allowed to run on a mobile device as well as on a desktop computer or laptop. If you use `display: flex` with your elements, is not necessary to add any fixed width or height. This is true for ALL other fixed widths that you have along with your CSS code. Please fix this